Ska/report_ranges/__init__.py

```diff
@@ -0,0 +1,5 @@
+import ska_helpers
+
+from .report_ranges import *
```

Comment:
And I see our standard flake8 is unhappy with this.  Is that an OK exception to add or do we have a new strategy?

Reply:
The error is about it being imported and not used. In that sense, this is an OK exception, in my opinion.

I think `import *` is not a good thing, although this might be an OK exception about that also.

Reply:
Right. I *understand* the flake8 warnings, it just seems like this is always the right thing if you are using a dummy `__init__.py` to import everything from the real package at this same level.  Unless we have a new plan.  I can certainly call out the methods, that just wasn't what we were doing, say from Ska.Shell.

If we decide to leave the code as-is I'm just not sure if we #noqa this or add 403 to the flake8 exceptions.

Reply:
I went for the #noqa